Modules/CalcOffence-3_0.lua

```diff
@@ -2727,4 +2796,5 @@ function calcs.offence(env, actor, activeSkill)
 			t_insert(breakdown.ImpaleDPS, s_format("= %.1f", output.ImpaleDPS))
 		end
 	end
+	return { Cooldown = output.Cooldown, CastTime = output.WarcryCastTime }
```

Comment:
Since this return {} is on a generic and widely used function, the things it returns should be generic. The cooldown one is fine since the return is called "Cooldown" and its value is output.Cooldown, However, "CastTime" should be changed to something specific like "WarcryTime" so that the "CastTime" can potentially be used more generically in the future.

Reply:
Agreed.... will do. I was going to do that actually and then thought to myself "no one else uses it currently, so no one will care". But you are right. Will fix.